WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -195,6 +207,16 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
         }
     }
 
+    private fun MeFragmentBinding.initUnifiedAboutUiState() {
+        setRecommendLoadingState(false)
```

Comment:
We shouldn't need to call `setRecommendLoadingState` here.

Reply:
font is appearing greyed out, without that.  Strange but true.  Something to do with shimmer layout and setup perhaps.

Reply:
I see. I think that's one more reason not to reuse the layout for this item.